replaying-share-kotlin/src/main/kotlin/com/jakewharton/rx/ReplayingShare.kt

```diff
@@ -25,8 +25,12 @@ import io.reactivex.Observable
  * value from the upstream observable *only* when one or more downstream observers are connected.
  * This allows expensive upstream observables to be shut down when no one is observing while also
  * replaying the last value seen by *any* observer to new ones.
+ *
+ * @param defaultValue the initial value to be cached
  */
-fun <T> Observable<T>.replayingShare(): Observable<T> = compose(ReplayingShare.instance<T>())
+fun <T> Observable<T>.replayingShare(defaultValue: T? = null): Observable<T> {
```

Comment:
Need to add `@JvmOverloads` to both to maintain binary compatibility, unfortunately.